### PR TITLE
Chapter1/homework/4

### PR DESCRIPTION
**Pull request by Precigus**
Finished homework 4.
Wasn't able to make all the tests pass though. Feedback on that would be nice

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208&bid=16859225)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            }
+
+            return result;
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+
+            var style = NumberStyles.Float;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Add the 'const' modifier to 'style'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            {
+                weightError = true;
+            }
+
+            if (height <= 0)
+            {
+                heightError = true;
+            }
+
+            if (!weightError && !heightError) return (weight / (height * height));
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+
+            if (string.IsNullOrEmpty(input))
+            {
+                input = "-";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this useless assignment to local variable 'input'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208)

Comment (Precigus) [reply Src/BootCamp.Chapter/Lesson3.cs]: Task requirement:

Validate name input:
Empty string for name -> return "-".
Print error message "Name cannot be empty." in a new line.

So just ignore this requirement then, right?

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5130208&bid=16859225)

**Pull request closed without merging**